packages/pilot/src/containers/TransactionsList/Charts.js

```diff
@@ -41,9 +41,8 @@ const renderLegend = (legendKey) => {
       <Legend
         acronym={status.acronym}
```

Comment:
Vi que na área do gráfico ainda temos algumas legendas que estão utilizando o acrônimo. Vamos manter assim?
Por exemplo: `CA`, `SU` e `IN`.
![image](https://user-images.githubusercontent.com/46823713/68124954-459bfb80-feef-11e9-9541-271271c57910.png)
```suggestion
        acronym={status.text}
```

Reply:
removendo a proeprty acronym como comentei no review resolve esse problema, como tudo vai ser texto completo agora deveriamos matar a prop acronym da model